sap_hana/assets/dashboards/overview.json

```diff
@@ -0,0 +1,586 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "SAP HANA Overview",
+    "created": "2019-10-23T22:03:16.131906+00:00",
+    "created_by": {
+        "access_role": "st",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": false,
+        "name": "Datadog",
+        "role": null,
+        "verified": true
+    },
+    "description": null,
```

Comment:
The description is missing. E.g.

https://github.com/DataDog/integrations-core/blob/fb381f12055cbc76d079bb259c04c494651c5fc7/ibm_db2/assets/dashboards/overview.json#L7

https://github.com/DataDog/integrations-core/blob/39529232e021bbef72244fe701a3769b64a7dc29/mapr/assets/dashboards/mapr_overview.json#L7